### PR TITLE
Refactor PointDetail contoller

### PR DESCRIPTION
**Pull request by SepsiLaszlo**

### CODE REVIEW

File: app/models/entry_request.rb [hunk 1]
```diff
@@ -41,4 +42,13 @@ def self.remove_justifications
...
   end
+
+  private
+
+  def correct_user
+    return if user.member_of?(evaluation.group)
+
+    errors.add(:user, "user is not a member of the current group")
+  end
+
```
Comment (falusi94) [inline app/models/entry_request.rb:53]: Please remove the empty line.

File: app/controllers/entry_requests_controller.rb [hunk 2]
```diff
@@ -2,13 +2,23 @@ class EntryRequestsController < ApplicationController
...
   def update
-    user = User.find params[:user_id]
+    begin
+      create_or_update_point_request
+    rescue ActiveRecord::RecordInvalid, RecordNotFound
+      return head :unprocessable_entity
+    end
+
+    head :ok
+  end
```
Comment (falusi94) [inline app/controllers/entry_requests_controller.rb:12]: Here you can leave the `begin`/`end` also.
```suggestion
    create_or_update_point_request
    head :ok  
  rescue ActiveRecord::RecordInvalid, RecordNotFound
    head :unprocessable_entity
  end
```

**Review by falusi94 — COMMENTED** [review]
What do you think about a more descriptive commit message like `Validate entry request's user on create`? Or even, not necessarily, it could be split into two commits like:
- Refactor EntryRequestsController
- Validate entry request's user on create

if you prefer so. Separation of refactoring and behavior changes, and descriptive commit messages make easier to understand the evolution of the codebase, finding possible bugs, or reverting an invalid change.

**Comment by SepsiLaszlo** [comment]
I updated the files! I'll try to find more descriptive commit messages!

**Review by falusi94 — APPROVED** [review]

**Pull request merged**